app/scripts/modules/core/src/pipeline/config/stages/rollbackCluster/RollbackClusterExecutionDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps } from 'core/pipeline/config/stages/core';
+import { HelpField } from 'core/help/HelpField';
+import { NgReact } from 'core/reactShims';
+import { StageExecutionLogs, StageFailureMessage } from 'core/pipeline/details';
+
+export function RollbackClusterExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { AccountTag } = NgReact;
+  const { stage } = props;
+
+  const imagesToRestore = stage.context.imagesToRestore && stage.context.imagesToRestore.map((entry: any) => {
+    const imageDetails = entry.buildNumber ? '#' + entry.buildNumber : entry.image;
+    const method = (entry.rollbackMethod === 'EXPLICIT') ? 'previous server group' : 'new server group';
+    const helpId = 'cluster.rollback.' + entry.rollbackMethod.toLowerCase();
+    return (
+      <div key={entry.region}>
+        <dt>
```

Comment:
These should probably be wrapped in a `<dl>` - not sure how the browser handles dd/dt tags sitting outside a dl...

Reply:
👍